cmd/operator/deploy/rule-evaluator/02-service-account.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: gmp-system
```

Comment:
Can you remove any `namespace` specifications? Again, we'd like to keep this manifest namespace-agnostic as it can be run in any namespace a user wishes via `kubectl apply -n <namespace>`.

It's here as well as in the role/role-bindings.